crates/gen/src/tables/attribute.rs

```diff
@@ -33,42 +36,62 @@ impl Attribute {
             AttributeType::MemberRef(method) => (reader.blob(method.0, 2), reader.blob(self.0, 2)),
         };
 
-        values.read_u16();
-        sig.read_unsigned();
-        let count = sig.read_unsigned();
-        sig.read_unsigned();
+        let prolog = values.read_u16();
+        assert!(prolog == 0x0001, "CustomAttribute Prolog must be 0x0001"); // Required by spec.
```

Comment:
Let's `debug_assert` this for now. There's a bunch of validation work we may need to do, but I'd like to apply that consistently.